roles/dotnet/meta/main.yml

```diff
@@ -3,7 +3,7 @@ galaxy_info:
   author: Israel Ogbole (https://galaxy.ansible.com/iogbole)
```

Comment:
Include you as an author here please.. also, include you as an author in both galaxy.yaml file